developer-docs-site/docs/reference/glossary.md

```diff
@@ -187,6 +187,15 @@ then there is a guarantee that T_N will never be included in the blockchain.
 
 - **Fullnodes** are clients that ensure data are stored up-to-date on the network. They replicate blockchain state and transactions from other fullnodes and validator nodes.
 
+### Fungible Asset
+
+- A **fungible asset** is an asset, such as a currency, share, in-game resource, etc., that can be interchanged with another identical asset without any loss in its value. For example, APT is fungible because you can exchange one APT for another.
+- Follow the [Digital Asset Standards](../standards/index.md#digital-asset-standards) to create fungible assets on the blockchain, including tokens and coins. 
```

Comment:
tokens usually means nft in our docs, which is not FA.

Reply:
But we also have docs where we refer to FTs as tokens. [AIP-21](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-21.md) uses terms like "tokenization" and "tokenized" to describe FAs. That being said, I'll remove the last bit from the second bullet point so that we can let the context in each doc decide.